integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -884,6 +884,10 @@ public static void createDockerRegistrySecret(
     domainMap.put("exposeAdminNodePort", true);
     domainMap.put("adminNodePort", 30700 + number);
     domainMap.put("t3ChannelPort", 30000 + number);
+    if (System.getenv("LB_TYPE") != null && System.getenv("LB_TYPE").equalsIgnoreCase("VOYAGER")) {
+      domainMap.put("voyagerWebPort", 30344 + number);
+      logger.info("For this domain voyagerWebPort is set to: 30344 + " + number);
```

Comment:
logger.info("For this domain voyagerWebPort is set to:  " + (30344 + number));

Reply:
The reason I separate "30344" and "number" is to make "number" easy to be detected.

Reply:
Maggie, seems two things here we may need to take care of:

1.  Seems that you don't have latest version. 
based on 
https://github.com/oracle/weblogic-kubernetes-operator/blob/develop/integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

in line 1350, Domain.java, it looks like:
   this.voyager =
        (System.getenv("LB_TYPE") != null && System.getenv("LB_TYPE").equalsIgnoreCase("VOYAGER"))
            || (inputDomainMap.containsKey("LB_TYPE")
                && ((String) inputDomainMap.get("LB_TYPE")).equalsIgnoreCase("VOYAGER"));

 2. still in line 1350, Domain.java,

   this.voyager =
        (System.getenv("LB_TYPE") != null && System.getenv("LB_TYPE").equalsIgnoreCase("VOYAGER"))
            || (inputDomainMap.containsKey("LB_TYPE")
                && ((String) inputDomainMap.get("LB_TYPE")).equalsIgnoreCase("VOYAGER"));

You can see here, we have two conditions in order to switch to use Voyager. However, in above line, we have the second condition below missing that may break Sticky Session tests since Treafik doesn't work for Sticky Session feature

(inputDomainMap.containsKey("LB_TYPE")
                && ((String) inputDomainMap.get("LB_TYPE")).equalsIgnoreCase("VOYAGER"));

Reply:
Done. After change Jenkins passed at http://wls-jenkins.us.oracle.com/view/weblogic-operator/job/weblogic-kubernetes-operator-javatest/1615